### PR TITLE
Disable highlighting for long or wide diffs

### PR DESCRIPTION
**Pull request by kumar303**
Fixes https://github.com/mozilla/addons-code-manager/issues/942

~~I chose not to add any UI feedback about the disabled highlighting only because this is probably a temporary feature.~~ (there is UI feedback now)

Screenshot 2019-07-11 12 01 09
Screenshot 2019-07-11 12 01 40

### CODE REVIEW

**Comment by willdurand** [comment]
> I chose not to add any UI feedback about the disabled highlighting only because this is probably a temporary feature.

Do you think that we'll always be able to highlight code with a web worker? I am not sure and I think a simple message somewhere would be very useful.

**Comment by kumar303** [comment]
ok, I can add a UI message. I guess maybe we'll always disable highlighting for minified diffs but I'm not sure.

File: src/components/DiffView/index.spec.tsx [hunk 1]
```diff
@@ -572,6 +618,44 @@ describe(__filename, () => {
     expect(getWidgetNodes(widgets).length).toEqual(0);
   });
 
+  it('enables syntax highlighting for diffs when possible', () => {
+    const _tokenize = jest.fn(tokenize);
+    const root = renderWithLinterProvider({
+      _diffCanBeHighlighted: jest.fn(() => true),
```
Comment (willdurand) [inline src/components/DiffView/index.spec.tsx]: that or `jest.fn().mockReturnValue(true)`

Comment (kumar303) [reply src/components/DiffView/index.spec.tsx]: It hurts my fingers to type all that 😆 jk

**Pull request merged**